app/models/job-link.ts

```diff
@@ -0,0 +1,162 @@
+import _ from 'lodash';
+import { Transaction } from 'util/db';
+import { removeEmptyProperties } from 'util/object';
+import Record from './record';
+
+interface BaseJobLink {
+  id?: number;
+  jobID?: string;
+  href: string;
+  type?: string;
+  title?: string;
+  rel?: string;
+  createdAt?: Date;
+  updatedAt?: Date;
+}
+
+interface JobLinkRecord extends BaseJobLink {
+  bbox?: string;
+  temporalStart?: Date;
+  temporalEnd?: Date;
+}
+
+/**
+ * For the constructor we can take either a JobLink or a JobLinkRecord.
+ * When created from the database it will be a JobLinkRecord, but when
+ * constructed outside of the database it will likely be a JobLink
+ * which has a different representation for the bbox and temporal.
+ * Specifying JobLink | JobLinkRecord as the type causes many Typescript
+ * errors so we use this interface to avoid that.
+ */
+
+export interface JobLinkOrRecord extends BaseJobLink {
+  bbox?: string | number[];
```

Comment:
I think we should just pick one `string` or `number[]` for internal use and convert to the other as needed for responses, etc.

Reply:
Similar to the temporal and temporalStart/temporalEnd above. That particular interface allows either, but generally we use number[].